src/newfight.hpp

```diff
@@ -197,9 +197,11 @@ struct melee_combat_data {
     if (weapon) {
       if (ranged_combat_mode) {
         if (does_weapon_have_bayonet(weapon)) {
+          power += 2;
```

Comment:
Why are we adding 2 power? Can you toss in a comment with the source reference?

Reply:
Done. Rifle-butts do +2, pistol whipping is only +1, no differentiation between the two in code atm.